src/json/firebolt-specification.json

```diff
@@ -137,6 +137,81 @@
     },
     "xrn:firebolt:capability:usergrant:pinchallenge": {
       "level": "should"
+    },
+    "xrn:firebolt:capability:protocol:bluetoothle": {
+      "level": "could",
+      "use": {
+        "public": true,
+        "negotiable": false
+      },
+      "manage": {
+        "public": false,
+        "negotiable": false
+      },
+      "provide": {
+        "public": false,
+        "negotiable": false
+      }
+    },
+    "xrn:firebolt:capability:protocol:rf4ce": {
+      "level": "could",
+      "use": {
+        "public": true,
+        "negotiable": false
+      },
+      "manage": {
+        "public": false,
+        "negotiable": false
+      },
+      "provide": {
+        "public": false,
+        "negotiable": false
+      }
+    },
+    "xrn:firebolt:capability:accessory:bluetoothle-remote": {
+      "level": "could",
+      "use": {
+        "public": true,
```

Comment:
Do we definitely need accessory type capabilities? I could imagine use cases, but until we actually have them, maybe leave this out? If Bluetooth remotes just use the Bluetooth keyboard profile, then supporting bluetooth means bluetooth remotes are inherently supported, but I don't actually know if that's how they work.